### ISSUE TITLE
Broken go get

### ISSUE THREAD

**Issue by bernardolm** [julienschmidt/go-http-routing-benchmark#81]
After run a `go get -u -v` I got:

```bash
➜ go get -u -v
github.com/julienschmidt/go-http-routing-benchmark (download)
github.com/ant0ine/go-json-rest (download)
github.com/astaxie/beego (download)
github.com/bmizerany/pat (download)
github.com/dimfeld/httptreemux (download)
github.com/emicklei/go-restful (download)
github.com/gin-gonic/gin (download)
github.com/gin-contrib/sse (download)
github.com/golang/protobuf (download)
github.com/ugorji/go (download)
Fetching https://gopkg.in/go-playground/validator.v8?go-get=1
Parsing meta tags from https://gopkg.in/go-playground/validator.v8?go-get=1 (status code 200)
get "gopkg.in/go-playground/validator.v8": found meta tag get.metaImport{Prefix:"gopkg.in/go-playground/validator.v8", VCS:"git", RepoRoot:"https://gopkg.in/go-playground/validator.v8"} at https://gopkg.in/go-playground/validator.v8?go-get=1
gopkg.in/go-playground/validator.v8 (download)
Fetching https://gopkg.in/yaml.v2?go-get=1
Parsing meta tags from https://gopkg.in/yaml.v2?go-get=1 (status code 200)
get "gopkg.in/yaml.v2": found meta tag get.metaImport{Prefix:"gopkg.in/yaml.v2", VCS:"git", RepoRoot:"https://gopkg.in/yaml.v2"} at https://gopkg.in/yaml.v2?go-get=1
gopkg.in/yaml.v2 (download)
github.com/mattn/go-isatty (download)
github.com/go-macaron/macaron (download)
github.com/Unknwon/com (download)
github.com/go-macaron/inject (download)
Fetching https://golang.org/x/crypto/pbkdf2?go-get=1
Parsing meta tags from https://golang.org/x/crypto/pbkdf2?go-get=1 (status code 200)
get "golang.org/x/crypto/pbkdf2": found meta tag get.metaImport{Prefix:"golang.org/x/crypto", VCS:"git", RepoRoot:"https://go.googlesource.com/crypto"} at https://golang.org/x/crypto/pbkdf2?go-get=1
get "golang.org/x/crypto/pbkdf2": verifying non-authoritative meta tag
Fetching https://golang.org/x/crypto?go-get=1
Parsing meta tags from https://golang.org/x/crypto?go-get=1 (status code 200)
golang.org/x/crypto (download)
Fetching https://gopkg.in/ini.v1?go-get=1
Parsing meta tags from https://gopkg.in/ini.v1?go-get=1 (status code 200)
get "gopkg.in/ini.v1": found meta tag get.metaImport{Prefix:"gopkg.in/ini.v1", VCS:"git", RepoRoot:"https://gopkg.in/ini.v1"} at https://gopkg.in/ini.v1?go-get=1
gopkg.in/ini.v1 (download)
github.com/go-martini/martini (download)
github.com/codegangsta/inject (download)
github.com/go-playground/lars (download)
github.com/go-playground/form (download)
github.com/gorilla/websocket (download)
github.com/go-zoo/bone (download)
github.com/gocraft/web (download)
github.com/gorilla/mux (download)
github.com/julienschmidt/httprouter (download)
github.com/labstack/echo (download)
github.com/labstack/gommon (download)
github.com/mattn/go-colorable (download)
github.com/valyala/fasttemplate (download)
Fetching https://golang.org/x/crypto/acme/autocert?go-get=1
Parsing meta tags from https://golang.org/x/crypto/acme/autocert?go-get=1 (status code 200)
get "golang.org/x/crypto/acme/autocert": found meta tag get.metaImport{Prefix:"golang.org/x/crypto", VCS:"git", RepoRoot:"https://go.googlesource.com/crypto"} at https://golang.org/x/crypto/acme/autocert?go-get=1
get "golang.org/x/crypto/acme/autocert": verifying non-authoritative meta tag
Fetching https://golang.org/x/crypto/acme?go-get=1
Parsing meta tags from https://golang.org/x/crypto/acme?go-get=1 (status code 200)
get "golang.org/x/crypto/acme": found meta tag get.metaImport{Prefix:"golang.org/x/crypto", VCS:"git", RepoRoot:"https://go.googlesource.com/crypto"} at https://golang.org/x/crypto/acme?go-get=1
get "golang.org/x/crypto/acme": verifying non-authoritative meta tag
github.com/lunny/log (download)
github.com/lunny/tango (download)
github.com/mailgun/route (download)
github.com/vulcand/predicate (download)
github.com/gravitational/trace (download)
github.com/jonboulle/clockwork (download)
github.com/sirupsen/logrus (download)
Fetching https://golang.org/x/crypto/ssh/terminal?go-get=1
Parsing meta tags from https://golang.org/x/crypto/ssh/terminal?go-get=1 (status code 200)
get "golang.org/x/crypto/ssh/terminal": found meta tag get.metaImport{Prefix:"golang.org/x/crypto", VCS:"git", RepoRoot:"https://go.googlesource.com/crypto"} at https://golang.org/x/crypto/ssh/terminal?go-get=1
get "golang.org/x/crypto/ssh/terminal": verifying non-authoritative meta tag
Fetching https://golang.org/x/sys/unix?go-get=1
Parsing meta tags from https://golang.org/x/sys/unix?go-get=1 (status code 200)
get "golang.org/x/sys/unix": found meta tag get.metaImport{Prefix:"golang.org/x/sys", VCS:"git", RepoRoot:"https://go.googlesource.com/sys"} at https://golang.org/x/sys/unix?go-get=1
get "golang.org/x/sys/unix": verifying non-authoritative meta tag
Fetching https://golang.org/x/sys?go-get=1
Parsing meta tags from https://golang.org/x/sys?go-get=1 (status code 200)
golang.org/x/sys (download)
Fetching https://golang.org/x/net/context?go-get=1
Parsing meta tags from https://golang.org/x/net/context?go-get=1 (status code 200)
get "golang.org/x/net/context": found meta tag get.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at https://golang.org/x/net/context?go-get=1
get "golang.org/x/net/context": verifying non-authoritative meta tag
Fetching https://golang.org/x/net?go-get=1
Parsing meta tags from https://golang.org/x/net?go-get=1 (status code 200)
golang.org/x/net (download)
github.com/mikespook/possum (download)
Fetching https://gopkg.in/fsnotify.v1?go-get=1
Parsing meta tags from https://gopkg.in/fsnotify.v1?go-get=1 (status code 200)
get "gopkg.in/fsnotify.v1": found meta tag get.metaImport{Prefix:"gopkg.in/fsnotify.v1", VCS:"git", RepoRoot:"https://gopkg.in/fsnotify.v1"} at https://gopkg.in/fsnotify.v1?go-get=1
gopkg.in/fsnotify.v1 (download)
Fetching https://golang.org/x/net/websocket?go-get=1
Parsing meta tags from https://golang.org/x/net/websocket?go-get=1 (status code 200)
get "golang.org/x/net/websocket": found meta tag get.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at https://golang.org/x/net/websocket?go-get=1
get "golang.org/x/net/websocket": verifying non-authoritative meta tag
github.com/naoina/denco (download)
github.com/naoina/kocha-urlrouter (download)
github.com/pilu/traffic (download)
github.com/pilu/config (download)
github.com/plimble/ace (download)
github.com/plimble/sessions (download)
github.com/oxtoacart/bpool (download)
github.com/satori/go.uuid (download)
github.com/tinylib/msgp (download)
github.com/philhofer/fwd (download)
github.com/plimble/utils (download)
github.com/rcrowley/go-tigertonic (download)
github.com/rcrowley/go-metrics (download)
github.com/revel/revel (download)
github.com/revel/config (download)
github.com/revel/log15 (download)
github.com/inconshreveable/log15 (download)
Fetching https://gopkg.in/stack.v0?go-get=1
Parsing meta tags from https://gopkg.in/stack.v0?go-get=1 (status code 200)
get "gopkg.in/stack.v0": found meta tag get.metaImport{Prefix:"gopkg.in/stack.v0", VCS:"git", RepoRoot:"https://gopkg.in/stack.v0"} at https://gopkg.in/stack.v0?go-get=1
gopkg.in/stack.v0 (download)
github.com/revel/pathtree (download)
Fetching https://gopkg.in/natefinch/lumberjack.v2?go-get=1
Parsing meta tags from https://gopkg.in/natefinch/lumberjack.v2?go-get=1 (status code 200)
get "gopkg.in/natefinch/lumberjack.v2": found meta tag get.metaImport{Prefix:"gopkg.in/natefinch/lumberjack.v2", VCS:"git", RepoRoot:"https://gopkg.in/natefinch/lumberjack.v2"} at https://gopkg.in/natefinch/lumberjack.v2?go-get=1
gopkg.in/natefinch/lumberjack.v2 (download)
github.com/xeonx/timeago (download)
Fetching https://gopkg.in/fsnotify/fsnotify.v1?go-get=1
Parsing meta tags from https://gopkg.in/fsnotify/fsnotify.v1?go-get=1 (status code 200)
get "gopkg.in/fsnotify/fsnotify.v1": found meta tag get.metaImport{Prefix:"gopkg.in/fsnotify/fsnotify.v1", VCS:"git", RepoRoot:"https://gopkg.in/fsnotify/fsnotify.v1"} at https://gopkg.in/fsnotify/fsnotify.v1?go-get=1
gopkg.in/fsnotify/fsnotify.v1 (download)
github.com/robfig/pathtree (download)
github.com/typepress/rivet (download)
github.com/ursiform/bear (download)
github.com/vanng822/r2router (download)
github.com/zenazn/goji (download)
Fetching https://goji.io?go-get=1
Parsing meta tags from https://goji.io?go-get=1 (status code 200)
get "goji.io": found meta tag get.metaImport{Prefix:"goji.io", VCS:"git", RepoRoot:"https://github.com/goji/goji"} at https://goji.io?go-get=1
goji.io (download)
Fetching https://goji.io/internal?go-get=1
Parsing meta tags from https://goji.io/internal?go-get=1 (status code 200)
get "goji.io/internal": found meta tag get.metaImport{Prefix:"goji.io", VCS:"git", RepoRoot:"https://github.com/goji/goji"} at https://goji.io/internal?go-get=1
get "goji.io/internal": verifying non-authoritative meta tag
Fetching https://goji.io?go-get=1
Parsing meta tags from https://goji.io?go-get=1 (status code 200)
Fetching https://goji.io/pat?go-get=1
Parsing meta tags from https://goji.io/pat?go-get=1 (status code 200)
get "goji.io/pat": found meta tag get.metaImport{Prefix:"goji.io", VCS:"git", RepoRoot:"https://github.com/goji/goji"} at https://goji.io/pat?go-get=1
get "goji.io/pat": verifying non-authoritative meta tag
Fetching https://goji.io/pattern?go-get=1
Parsing meta tags from https://goji.io/pattern?go-get=1 (status code 200)
get "goji.io/pattern": found meta tag get.metaImport{Prefix:"goji.io", VCS:"git", RepoRoot:"https://github.com/goji/goji"} at https://goji.io/pattern?go-get=1
get "goji.io/pattern": verifying non-authoritative meta tag
github.com/julienschmidt/go-http-routing-benchmark
# github.com/julienschmidt/go-http-routing-benchmark
./routers.go:1151:33: rc.Controller.Request.RequestURI undefined (type *revel.Request has no field or method RequestURI)
./routers.go:1167:27: cannot use r (type *http.Request) as type revel.ServerRequest in argument to revel.NewRequest:
	*http.Request does not implement revel.ServerRequest (missing Get method)
./routers.go:1168:28: cannot use w (type http.ResponseWriter) as type revel.ServerResponse in argument to revel.NewResponse:
	http.ResponseWriter does not implement revel.ServerResponse (missing Get method)
./routers.go:1169:30: too many arguments in call to revel.NewController
	have (*revel.Request, *revel.Response)
	want (revel.ServerContext)
./routers.go:1171:6: req.Websocket undefined (type *revel.Request has no field or method Websocket, but does have WebSocket)
./routers.go:1179:23: invalid type assertion: resp.Out.(io.Closer) (non-interface type revel.OutResponse on left)
./routers.go:1218:33: not enough arguments in call to revel.NewRoute
	have (string, string, string, string, string, number)
	want (*revel.Module, string, string, string, string, string, int)
./routers.go:1223:14: cannot use "github.com/robfig/pathtree".New() (type *"github.com/robfig/pathtree".Node) as type *"github.com/revel/pathtree".Node in assignment
./routers.go:1244:25: not enough arguments in call to revel.NewRoute
	have (string, string, string, string, string, number)
	want (*revel.Module, string, string, string, string, string, int)
```

**Comment by bernardolm** [comment]
Revel is breaking the fun. After remove it, the party go all night long ;)

**Comment by julienschmidt** [comment]
Fixed by 052672f2944d5f4205b707a55ab1764ffa321f23
